src/Tracker/Source/LimeSurvey5m00FieldMap.php

```diff
@@ -78,170 +80,18 @@ protected function _setMap(): void
             WHERE g.sid = ? AND qt.language = ? AND gt.language = ? AND q.parent_qid = 0
             ORDER BY g.group_order, q.question_order, sq.scale_id DESC, sq.question_order";
 
-        $rows = $this->lsResultFetcher->fetchAll($sql, [$this->sourceSurveyId, $this->language, $this->language]);
```

Comment:
Klopt het dat al deze code verwijderd is? Meestal verwacht ik daar wel een mededeling over in de commit message..